src/test/java/core/basesyntax/service/convertator/DataConvertorImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service.convertator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class DataConvertorImplTest {
+    private static final String CORRECT_DATA_FOR_CONVERTATION = "type,fruit,quantity"
+            + System.lineSeparator()
+            + "b,banana,20"
+            + System.lineSeparator()
+            + "b,apple,100"
+            + System.lineSeparator()
+            + "s,banana,100"
+            + System.lineSeparator()
+            + "p,banana,13"
+            + System.lineSeparator()
+            + "r,apple,10"
+            + System.lineSeparator()
+            + "p,apple,20"
+            + System.lineSeparator()
+            + "p,banana,5"
+            + System.lineSeparator()
+            + "s,banana,50";
```

Comment:
extract this to .csv and move it to test/resource

write parametrized test or something like that